### ISSUE TITLE
Better way to provide better documentation/help from within Stacks

### ISSUE THREAD

**Issue by joeworkman** [yourhead/s3#774]
I don't know exactly what to do here but, I would love to see a better way to provide documentation to customers from within the Stacks UI.

**Comment by yourhead** [comment]
me too.

but it's hard to come up with specifics here. 

seen any other apps that handle detailed docs/help really well? i need some ideas on this one. i haven't seen anyone attack this in a nice way. most documentation kind of sucks.

**Comment by joeworkman** [comment]
Could we somehow leverage the MacOS help system? Ship some help files inside the stack?

**Comment by yourhead** [comment]
Yes, probably. But the macOS help system isn't very contextual or interactive. It's kind of a 🔥💩 --  seems like it might be nice to have contextual bubbles that pop up or something along those lines.

Or maybe a way to have a master checkbox that switches inline help on/off?

Or how 'bout a way to have bubbles that guide you through setup when you drop in a stack -- but have the "Don't show this again" checkbox at the bottom?

**Comment by yourhead** [comment]
I'm just sort of keeping my eyes open when using other tools -- especially on the first-launch behavior of small apps. That's probably the closest analog to a big stack.
